bindings/Sofa/src/SofaPython3/Sofa/Core/Binding_Node.cpp

```diff
@@ -185,6 +188,26 @@ py::object getObject(Node &n, const std::string &name, const py::kwargs& kwargs)
     return py::none();
 }
 
+void setFieldsFromPythonValues(Base* self, const py::kwargs& dict)
+{
+    // For each argument of the addObject functin we check if this is a
```

Comment:
```suggestion
    // For each argument of the addObject function we check if this is a
```
I think the sentence is not finished

Reply:
DIdn't had commited the suggestion ?